.travis.yml

```diff
@@ -3,25 +3,24 @@ sudo: false
 language: scala
 cache:
   directories:
-    - $HOME/.ivy2/cache
-    - $HOME/.sbt/boot/
+    - $HOME/.sbt
+    - $HOME/.ivy2
+    - $HOME/.m2
 
 jdk:
 - openjdk8
 
 matrix:
```

Comment:
Think the matrix could be simplified even more perhaps, to how we do it in: https://github.com/hmrc/play-language/blob/b43a6730fb98bc2479ad36a75db23614fef47c97/.travis.yml#L3-L11